go/internal/base/json_test.go

```diff
@@ -58,6 +63,21 @@ func TestExtractJSONFromMarkdown(t *testing.T) {
 			in:   "```json{\"a\":\n1}```\n```json\n{\"b\":\n1}```",
 			want: "{\"a\":\n1}",
 		},
+		{
+			desc: "yaml markdown",
+			in:   "```yaml\nkey: 1\nanother-key: 2```",
+			want: "```yaml\nkey: 1\nanother-key: 2```",
+		},
+		{
+			desc: "yaml + json markdown",
+			in:   "```yaml\nkey: 1\nanother-key: 2``` ```json\n{\"a\": 1}\n```",
+			want: "\n{\"a\": 1}\n",
```

Comment:
@apascal07 we're keeping the original string with the original new line characters.

Is it okay if we keep them or should we trim them?

Reply:
We should trim them down.